components/navbar.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
Since this is a component, remove this head content and link the CSS styles. And change html extension to php. Make sure to drop the screenshot of nav bar